config/initializers/strong_migrations.rb

```diff
@@ -1,5 +1,7 @@
+# frozen_string_literal: true
+
 # Mark existing migrations as safe
-StrongMigrations.start_after = 20220604103854
+StrongMigrations.start_after = 20_220_604_103_854
```

Comment:
This separation makes no sense (although I do know this is Rubocop's suggestion). We can just ignore this rule here:

```rb
StrongMigrations.start_after = 20220604103854 # rubocop:disable Style/NumericLiterals
```

Or if you'd prefer a sensible format:

```rb
StrongMigrations.start_after = 2022_06_04_10_38_54 # rubocop:disable Style/NumericLiterals
```

Or even not ignoring the rule and transforming it into a string

```rb
StrongMigrations.start_after = '20220604103854'
```

which IMO looks a bit better, since migration versions are not really integers, but dates.

Let me know what you think :)

Reply:
I think turning into a string looks the best

Reply:
Buuut turning it into a string created an error when running migrations, so I'm going to use the first option